SimplenoteUITests/NoteEditor.swift

```diff
@@ -57,7 +57,7 @@ class NoteEditor {
 
     class func swipeToPreview() {
         app.textViews.firstMatch.swipeLeft()
-        sleep(1)
+        sleep(5)
```

Comment:
At some point, it would be great to replace these `sleep()` calls with something that polls the UI looking for a criteria that can validate whether the element has loaded. Doing so would make the tests more robust.

Having said that, I acknowledge it's a challenging thing to do and sometime may require a lot of extra work on the production code to expose hooks we can use to validate the UI from here.

Reply:
I agree. This is a thing I'd like to address in one of the upcoming PRs. `sleep(1)` looks harmless and easy to justify, until the moment when it turns to `sleep(11)`. As always, here I did it to quickly unblock the 'main' task for the PR - which was the accounts handling.